File: Next_Layer_Boost_App-main/app/src/main/java/com/sis/clighteningboost/Activities/NodeIDResponse.java

package com.sis.clighteningboost.Activities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.os.Handler;

import com.sis.clighteningboost.R;

public class NodeIDResponse extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_node_idresponse);


        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {

                finish();
            }
        },2000);
    }
}

File: app/src/main/java/com/sis/clighteningboost/Activities/NodeIDResponse.kt
package com.sis.clighteningboost.Activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import com.sis.clighteningboost.R

class NodeIDResponse : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_node_idresponse)
        Handler().postDelayed({ finish() }, 2000)
    }
}